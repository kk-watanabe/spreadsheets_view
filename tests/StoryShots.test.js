import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "config/storybook",
  // element-uiで乱数付与しているため除外
  storyKindRegex: /^((?!.*?Tooltip).)*$/
});
