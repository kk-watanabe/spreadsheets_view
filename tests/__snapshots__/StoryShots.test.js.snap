// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atom/Button default 1`] = `
<button
  class="button button--dark"
  type="button"
>
  Hello!
</button>
`;

exports[`Storyshots Atom/Form/InputPart default 1`] = `
<input
  class="input-part"
  placeholder="Sample text"
  type="text"
/>
`;

exports[`Storyshots Common/Footer default 1`] = `
<div
  class="footer"
>
  <!---->
   
  <small
    class="footer__copyright"
  >
    Â© 2019 Style guide views.
  </small>
</div>
`;
